scripts/setup-test-registry

```diff
@@ -29,12 +29,8 @@ docker stop "$REGISTRY_CONTAINER_NAME" || true && docker rm "$REGISTRY_CONTAINER
 
 echo "Running $REGISTRY_CONTAINER_NAME"
 docker run -d --name="$REGISTRY_CONTAINER_NAME" -e SETTINGS_FLAVOR=local -p "127.0.0.1:51670:5000" "${REGISTRY_IMAGE}"
-
-# Make sure our images are pushed to it
-mirror_image() {
-  docker pull $1
-  mirror_local_image $@
-}
+# give the registry some seconds to get ready for pushes
+sleep 7
```

Comment:
Defining the function here seems to have taken just enough time for some of the slower instances (like inf1) to reliably push to this local repo. Adding the sleep to avoid random tcp and EOF errors when we move on to push images to the local registry at `127.0.0.1:51670`